6.analyze_features/scripts/1.anova.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import statsmodels.api as sm
+import tqdm
+from scipy.stats import levene
+
+# improt anova and tukeyhsd
+from statsmodels.formula.api import ols
+from statsmodels.stats.multicomp import pairwise_tukeyhsd
+
+# In[2]:
+
+
+def anova_function(features_df: pd.DataFrame, Metdata_column: str) -> pd.DataFrame:
+    """
+    This function will take in a dataframe and a metadata column and return the results of an anova and tukeyhsd test for each feature.
+
+
+    Parameters
+    ----------
+    features_df : pd.DataFrame
+        The dataframe containing the features with only one metadata column
+    Metdata_column : str
+        The name of the metadata column to be used for the anova test
+
+    Returns
+    -------
+    pd.DataFrame
+        A dataframe containing the results of the anova and tukeyhsd test for each feature
+    """
+
+    # get dfs
+    # split the df into three genotypes
```

Comment:
Recommend removing this code comment since there is no code below where you are splitting based on genotype.